### ISSUE TITLE
Strange cut in line of commits

### ISSUE THREAD

**Issue by pelmato** [phil294/GitLG#116]
Hello, new version (v0.1.22) has introduced strange cuts in the line of commits:
![gitlg-bug](https://github.com/user-attachments/assets/facfdc15-be39-431f-9966-36729085df2d)

For comparison here is the same history with Git Graph:
![git-graph-compare](https://github.com/user-attachments/assets/e134b92c-13e5-4a24-96a1-ad62044d44b9)

I would say that there is a cut before a new color, but this is just an assumption.

Hope this helps you.
Thanks for your great work!

**Comment by dimateos** [comment]
That might be related to line height?

Try increasing it:
![image](https://github.com/user-attachments/assets/888f6498-bfcc-466e-8491-0062d70451e6)

Also your current theme affects it:
![2024-11-28_(142529)_Code](https://github.com/user-attachments/assets/dcef8946-dd52-47ee-ae41-8f7f171280f8)

**Comment by phil294** [comment]
Yes they are theme-dependent since recently, interesting that some themes seem to not have a particular color set to it. And I'm open for further style changes, unfortunately alternating colors looks pretty lame pretty fast.

But what @pelmato was referring to in this issue is the gap between the colored branch lines at the very left, such as between yellow circle and orange circle.

I have a fix for that already and actually a few more fixes and comments on other issues that I made last week on another machine, that I haven't pushed/submitted yet... so some of the things you commented on today @dimateos might already be solved, sorry. Should have pushed this right away. I'll have a look at everything tomorrow or so.

**Comment by dimateos** [comment]
Oh I see the line cuts!

Regarding the lines and themes, if its controlled by the theme then the user can modify it in "workbench.colorCustomizations", do you know which is the color id used?

---

No problem about the stuff I commented on yesterday! At least there are actionable issues / places to discuss.
Take your time, -zero pressure etc ofc
I myself will check from time to time

Cheers

**Comment by phil294** [comment]
Yes, pretty sure that's sideBarSectionHeader-border.

(related: #115)

**Comment by dimateos** [comment]
Yep its `"sideBarSectionHeader.border"` (note the dot, not dash), you can put it inside `workbench.colorCustomizations"` and override the theme:

![image](https://github.com/user-attachments/assets/948c22fd-91a2-4b60-a4d2-fbb21093e7f3)

**Comment by pelmato** [comment]
It looks fine now with version 1.30.0. Thank you @phil294 !
